data/api/client-server/capabilities.yaml

```diff
@@ -69,8 +69,7 @@ paths:
                 description: |-
                     The custom capabilities the server supports, using the
                     Java package naming convention.
-                additionalProperties:
-                  type: object
+                additionalProperties: true
```

Comment:
> capabilities.yaml: additional properties are actually more likely to be strings (but can also be objects)

are they?

looking at `/capabilities` on my server:

```json
{
  "capabilities": {
    "m.room_versions": {
      "default": "10",
      "available": {
        "1": "stable",
        "2": "stable",
        "3": "stable",
        "4": "stable",
        "5": "stable",
        "6": "stable",
        "org.matrix.msc2176": "unstable",
        "7": "stable",
        "8": "stable",
        "9": "stable",
        "org.matrix.msc3787": "unstable",
        "10": "stable",
        "org.matrix.msc2716v4": "unstable"
      },
      "org.matrix.msc3244.room_capabilities": {
        "knock": {
          "preferred": "7",
          "support": [
            "7",
            "8",
            "9",
            "org.matrix.msc3787",
            "10",
            "org.matrix.msc2716v4"
          ]
        },
        "restricted": {
          "preferred": "9",
          "support": [
            "8",
            "9",
            "org.matrix.msc3787",
            "10"
          ]
        }
      }
    },
    "m.change_password": {
      "enabled": true
    },
    "m.set_displayname": {
      "enabled": true
    },
    "m.set_avatar_url": {
      "enabled": true
    },
    "m.3pid_changes": {
      "enabled": true
    }
  }
}
```

... they are *all* objects.

Reply:
God knows what I smoked when writing that. Thanks for checking after me.

Reply:
```suggestion
                additionalProperties:
                  type: object
```